decidim-admin/app/controllers/concerns/decidim/admin/needs_admin_tos_accepted.rb

```diff
@@ -7,6 +7,7 @@ module NeedsAdminTosAccepted
       extend ActiveSupport::Concern
 
       included do
+        include UserRoleChecker
```

Comment:
```suggestion
      include UserRoleChecker
        
      included do
```
We can try to do the include here

Reply:
Done